demo/fit_a_line/train.py

```diff
@@ -13,26 +10,30 @@
 trainer_id = int(os.getenv("PADDLE_INIT_TRAINER_ID"))
 trainer_count = int(os.getenv("PADDLE_INIT_NUM_GRADIENT_SERVERS"))
 
-def recordio_reader_creator(path):
-    files = glob.glob(path)
-    files.sort()
-    flie_count = len(files)
-    files_current_train = []
+# TODO(helin): remove this once paddle.v2.reader.creator.recordio is
+# fixed.
+def recordio(paths, buf_size=100):
```

Comment:
Thanks! Sharding does not work correctly before https://github.com/PaddlePaddle/cloud/issues/319 and https://github.com/PaddlePaddle/cloud/issues/318 is fixed, can I merge this first and submit a follow up PR for sharding after these issues being resloved?